module/behaviour/planning/KickPlanner/src/KickPlanner.cpp

```diff
@@ -92,12 +100,20 @@ namespace behaviour {
                 }
             });
 
-            on<Trigger<Ball>, With<Field>, With<FieldDescription>, With<KickPlan>, With<Sensors>>().then(
-                [this](const Ball& ball,
-                       const Field& field,
-                       const FieldDescription& fd,
-                       const KickPlan& kickPlan,
-                       const Sensors& sensors) {
+            on<Trigger<ButtonMiddleDown>, Single>().then([this] { forcePlaying = true; });
```

Comment:
How do you turn it off? could we do something like `forcePlaying = !forcePlaying`?

Reply:
Middle button always plays. Turning it off is red button.